npm/README.md

```diff
@@ -1,18 +1,18 @@
-<h1 align="center">Infisical</h1>
+<h1 align="center">Infisical CLI</h1>
 <p align="center">
-  <p align="center"><b>The open-source secret management platform</b>: Sync secrets/configs across your team/infrastructure and prevent secret leaks.</p>
+  <p align="center"><b>The Infisical CLI is powerful command line tool that can be used to retrieve, modify, export and inject secrets into any process or application as environment variables.</b></p>
```

Comment:
typo: missing a -> `is a powerful command...`

but I would suggest making this top section more grabby and then using the bottom section to be more explicit/technical; maybe something like this?

"Embrace shift-left security with Infisical CLI and strengthen your DevSecOps practices by seamlessly managing secrets across your workflows, pipelines and applications."

Reply:
Implemented!